src/tree/SubscriptionTreeItem.ts

```diff
@@ -40,7 +41,15 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
 
     public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
         const client: WebSiteManagementClient = await createWebSiteClient([context, this]);
-        const staticWebApps: StaticSiteARMResource[] = await uiUtils.listAllIterator(client.staticSites.list());
+        let staticWebApps: StaticSiteARMResource[] = []
+
+        try {
+            staticWebApps = await uiUtils.listAllIterator(client.staticSites.list());
+        } catch (e) {
+            if (isRestError(e) && e.code === 'InvalidResourceType') {
```

Comment:
I'd recommend using `parseError` here. Then we don't need to create the `isRestError` function because checking if it's an `InvalidResourceType` is enough, right?

Also make sure we aren't relying on the error code casing. Something like: 

```ts
if (/InvalidResourceType/i.test(parseError(e).errorType)) {
```

Reply:
Thanks, I didn't know the power of `parseError`  😄 🚀